android/src/com/mapswithme/maps/location/LocationHelper.java

```diff
@@ -70,15 +70,27 @@
     @Override
     public void onTransit(boolean foreground)
     {
-      if (foreground)
+      // Note. onTransit(true) callback should be called when the application goes to foreground and
+      // onTransit(false) should be called when it goes to backgroud. According to Fabric in some cases
+      // this method could be called onTransit(false) without previous call onTransit(true).
+      // As a result we have an attempt to unregister |mReceiver| which was not registered before.
+      // That leads to IllegalArgumentException.
+      try
```

Comment:
I generally dislike the idea to catch all IllegalArgumentExeptions "just because we can." Please, close the CL and take a look at https://github.com/mapsme/omim/pull/4068